### DISCUSSION TITLE
Debug variants of Android libraries are being published (`kermit-android-debug`)

### DISCUSSION THREAD

**Discussion by alexvanyo** [touchlab/Kermit#427] [General]
Right now, both `kermit-android` and `kermit-android-debug` libraries are being published for Android. Looking through the code, it looks like the `debug` variant is identical to the `release` variant, so which version of the library is being used by a consumer shouldn't matter right now, but theoretically it could in the future. For example, theoretically `kermit-android-debug` could have a different default logging behavior than `kermit-android`.

Is publishing both variants intentional? If they are intended to behave identically, would only publishing the release version be clearer to consumers?

**Comment by kpgalligan** [comment]
Moving to discussions. I wouldn't expect to really dive into this in the near term. I'm reluctant to touch publishing in any way without significant review, research, testing, etc. Pretty much anything we've ever done WRT to publishing has caused somebody's build to break.

We have no foreseeable thoughts to change what happens in release vs debug as far as functionality, so the primary issue currently would be user confusion.
